core/new-gui/src/app/app.module.ts

```diff
@@ -225,56 +215,45 @@ registerLocaleData(en);
         throwNoTokenError: false,
       },
     }),
-    MatTooltipModule,
-    CustomNgMaterialModule,
     BrowserAnimationsModule,
     NgbModule,
-    NgbPopoverModule,
     RouterModule.forRoot([]),
     FileUploadModule,
     FormsModule,
     ReactiveFormsModule,
     FormlyModule.forRoot(TEXERA_FORMLY_CONFIG),
     FormlyMaterialModule,
-    FormlyMatDatepickerModule,
+    OverlayModule,
+    MatInputModule,
+    MatTabsModule,
     NzDatePickerModule,
     NzDropDownModule,
     NzButtonModule,
     NzAutocompleteModule,
     NzIconModule,
     NzFormModule,
-    NzImageModule,
     NzListModule,
     NzInputModule,
     NzPopoverModule,
-    NzCalendarModule,
-    NzMenuModule,
-    NzMessageModule,
     NzCollapseModule,
     NzToolTipModule,
-    NzCheckboxModule,
     NzTableModule,
-    NzModalModule,
     NzSelectModule,
-    NzSliderModule,
     NzSpaceModule,
     NzBadgeModule,
     NzUploadModule,
-    NzNoAnimationModule,
     NgxJsonViewerModule,
-    MatDialogModule,
+    NzMessageModule,
+    NzModalModule,
     NzCardModule,
-    NzStatisticModule,
     NzTagModule,
     NzPopconfirmModule,
     NzAvatarModule,
     DynamicModule,
     NzTabsModule,
-    NzTreeViewModule,
```

Comment:
I think the "EvaluateExpression" feature is using tree module. Can we keep this?

Reply:
We can easily add it back when we need it. If it's not directly called by any component, we should remove it.